app/src/components/main/grid/gridBody/gridRow/gridRow.jsx

```diff
@@ -81,6 +85,8 @@ export class GridRow extends Component {
     }
   }
 
+  onRowItemSelect = (e) => this.props.onSelectBulkItems(e, this.props.value);
```

Comment:
According to the comment above it can be:
`handleRowClick = (e) => this.props.onClickRow(this.props.value, e);`

Reply:
Fixed